agent/src/main/java/com/newrelic/agent/android/stores/SharedPrefsEventStore.java

```diff
@@ -45,7 +43,8 @@ public boolean store(AnalyticsEvent event) {
                 editor.putString(event.getEventUUID(), eventJson);
 
                 StatsEngine.SUPPORTABILITY.inc(MetricNames.SUPPORTABILITY_EVENT_SIZE_UNCOMPRESSED, eventJson.length());
-                return editor.commit();
+                editor.apply();
```

Comment:
This should be calling applyOrCommitEditor(), but I just noticed that method has been changed as well. There should be a commit() as a fail safe if apply throws an exception.

Reply:
i updated it.